esignet-default.properties

```diff
@@ -199,25 +226,22 @@ mosip.esignet.cache.store.individual-id=true
 mosip.esignet.cache.security.secretkey.reference-id=TRANSACTION_CACHE
 mosip.esignet.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 
-mosip.esignet.ida.vci-user-info-cache=userinfo
+mosip.esignet.cache.names=clientdetails,preauth,authenticated,authcodegenerated,userinfo,linkcodegenerated,linked,linkedcode,linkedauth,consented,authtokens,bindingtransaction,vcissuance,apiRateLimit,blocked
 
-mosip.esignet.cache.names=clientdetails,preauth,authenticated,authcodegenerated,userinfo,linkcodegenerated,linked,linkedcode,linkedauth,consented,authtokens,bindingtransaction,vcissuance
-
-spring.cache.type=redis
-spring.cache.cache-names=${mosip.esignet.cache.names}
-spring.redis.host=redis-master-0.redis-headless.redis.svc.cluster.local
-spring.redis.port=6379
+#spring.cache.type=redis
+#spring.cache.cache-names=${mosip.esignet.cache.names}
+#spring.redis.host=localhost
+#spring.redis.port=6379
 management.health.redis.enabled=false
-spring.redis.password=${redis.password}
 
 # 'simple' cache type is only applicable only for Non-Production setup
-#spring.cache.type=simple
+spring.cache.type=simple
 mosip.esignet.cache.key.hash.algorithm=SHA3-256
 
-# Cache size setup is applicable only for 'simple' cache type. 
+# Cache size setup is applicable only for 'simple' cache type.
 # Cache size configuration will not be considered with 'Redis' cache type
 mosip.esignet.cache.size={'clientdetails' : 200, \
-'preauth': 2000, \
+'preauth': 200, \
```

Comment:
Change the value to 2000 -> because for performance testing we will need higher preauth transaction time

Reply:
done